OpenRA.Mods.Cnc/Traits/Infiltration/Infiltrates.cs

```diff
@@ -119,7 +119,7 @@ public void ResolveOrder(Actor self, Order order)
 			if (!order.Queued)
 				self.CancelActivity();
 
-			self.SetTargetLine(target, Color.Red);
+			self.ShowTargetLines();
```

Comment:
I think it's wrong that this is called in trait code at all. It just causes the lines to appear at the wrong time when you queue commands. It should instead probably be called somewhere in `Activity.OuterTick`, although then you have to figure out how to deal with the engine/mod code separation.